megamek/src/megamek/common/actions/PhysicalAttackAction.java

```diff
@@ -211,8 +211,23 @@ protected static void setCommonModifiers(ToHitData toHit, IGame game,
         }
 
         // if we're spotting for indirect fire, add +1
-        if (ae.isSpotting() && !ae.getCrew().hasActiveCommandConsole()) {
-            toHit.addModifier(+1, "attacker is spotting for indirect LRM fire");
+        if (ae.isSpotting() && !ae.getCrew().hasActiveCommandConsole()) {           
+            //but if it spots by TAG it doesn't.
+            boolean isSpotByTag = false;//temporal boolean flag
```

Comment:
Comments that explain how the code works are usually good. Comments like this that explain how the language works are superfluous and add unnecessary noise.